packages/binding-file/src/file-client.ts

```diff
@@ -86,8 +86,9 @@ export default class FileClient implements ProtocolClient {
         error?: (error: Error) => void,
         complete?: () => void
     ): Promise<Subscription> {
-        error(new Error(`FileClient does not implement subscribe`));
-        return null;
+        return new Promise<Subscription>((resolve, reject) => {
```

Comment:
I prefer throwing directly. 🤔 is it possible?

Reply:
Not sure what you mean by "throwing directly" ?

I just aligned this method with what is done in all the other methods in this class
see https://github.com/eclipse-thingweb/node-wot/blob/master/packages/binding-file/src/file-client.ts

Reply:
I mean in theory you don't need to create the promise object and reject it. You can `throw new Error("...")` directly. But I see know that you used the same pattern that was in the file. So for consistency, I'm ok with this, we can fix it later.